source/stylesheets/refills/_laptop.scss

```diff
@@ -0,0 +1,101 @@
+.laptop {
+  $lid-background: darken(#fff, 5);
+  $inset-shadow: inset 0 0px 1px transparentize(#000, 0.8);
+  $laptop-size-ratio: 1.5;
+  $laptop-shrink-ratio: 0.5;
+  $bottom-width: 22em * $laptop-size-ratio;
+  $laptop-lid-width: 17em * $laptop-size-ratio;
+  $screen-height: 137px * $laptop-size-ratio;
+  $radius-outer: 15px * $laptop-size-ratio;
+  $radius-bottom: 15px * $laptop-size-ratio;
+  $padding-screen: 15px * $laptop-size-ratio;
+  $bottom-height: 15px * $laptop-size-ratio;
+  $bottom-detail-width: 3em * $laptop-size-ratio;
+  $bottom-detail-height: 4px * $laptop-size-ratio;
+  $shadow-height: 4px * $laptop-size-ratio;
+  $shadow-width: 25em * $laptop-size-ratio; 
+  $shadow-distance: -1px * $laptop-size-ratio;
+  $laptop-outlines: 1px solid #c0c0c0;
```

Comment:
Color literals like `#c0c0c0` should only be used in variable declarations; they should be referred to via variable everywhere else.